tools/project_manager/editproject.php

```diff
@@ -288,15 +278,9 @@ public function set_from_post()
             $projectid = get_projectID_param($_POST, 'projectid');
             $this->projectid = $projectid;
 
-            $ucep_result = user_can_edit_project($this->projectid);
-            if ($ucep_result == PROJECT_DOES_NOT_EXIST) {
-                return _("parameter 'projectid' is invalid: no such project").": '$this->projectid'";
-            } elseif ($ucep_result == USER_CANNOT_EDIT_PROJECT) {
+            $project = new Project($this->projectid);
```

Comment:
do we want to catch here and return the not found message like before with PROJECT_DOES_NOT_EXIT return response?

Reply:
Yes, good catch!